src/main/features/response/components/payment-intention/payment-plan.ts

```diff
@@ -0,0 +1,107 @@
+import * as express from 'express'
+
+import * as _ from 'lodash'
+
+import { Paths } from 'response/paths'
+import { Paths as PaymentIntentionPaths } from 'response/components/payment-intention/paths'
+
+import { GuardFactory } from 'response/guards/guardFactory'
+import { ErrorHandling } from 'shared/errorHandling'
+import { Form } from 'forms/form'
+import { DraftService } from 'services/draftService'
+import { User } from 'idam/user'
+import { DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { DefendantPaymentPlan } from 'response/form/models/defendantPaymentPlan'
+import { FormValidator } from 'forms/validation/formValidator'
+import { FeatureToggleGuard } from 'guards/featureToggleGuard'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { Claim } from 'claims/models/claim'
+import { createPaymentPlan } from 'common/calculate-payment-plan/paymentPlan'
+import { PaymentSchedule } from 'features/ccj/form/models/paymentSchedule'
+
+function mapFrequencyInWeeks (frequency: PaymentSchedule): number {
+  switch (frequency) {
+    case PaymentSchedule.EACH_WEEK:
+      return 1
+    case PaymentSchedule.EVERY_TWO_WEEKS:
+      return 2
+    case PaymentSchedule.EVERY_MONTH:
```

Comment:
a month doesn't map to 4 weeks, its 4.3333333 or `52 / 12`
Although I hear rumours of the practice direction being silly and putting 4.3 in

Reply:
This is what was implemented and signed off. Either we quietly accepted that bug or it is alright.  I would say it is out of scope go this PR (as code was just moved around) but we need to clarify requirement and raise bug that will result in another PR.

Reply:
Placeholder for story:
https://tools.hmcts.net/jira/browse/ROC-4173